src/api/wallet.js

```diff
@@ -0,0 +1,129 @@
+import axios from 'axios';
+import { host } from '../config';
+
+export const getWalletAddress = id => {
+  return axios.get(`${host}/api/v2/peatio/account/deposit_address/${id}`)
+    .then(response => response.data);
+};
+
+const getBalances = () => {
+  return axios.get(`${host}/api/v2/peatio/account/balances`)
+    .then(response => response.data);
+};
+
+const getCurrencies = () => {
+  return axios.get(`${host}/api/v2/peatio/public/currencies`)
+    .then(() => {//TODO remove after finishing API
```

Comment:
I think it's time to remove this part.
ideally we have an easy ability to embed fake data in redux.